management/src/main/java/com/bitwisekaizen/sdss/dto/IscsiTargetSpec.java

```diff
@@ -0,0 +1,49 @@
+package com.bitwisekaizen.sdss.dto;
+
+import java.util.List;
+
+/**
+ * Spec that describes the ISCSI target
+ */
+public class IscsiTargetSpec {
```

Comment:
Hm, kinda confusing as LioBackedIscsiAgent is not an agent.  It's additional information on the IscsiTarget.  In vSphere, there's a VirtualMachine and then within it, there's VirtualMachineConfigInfo that further describes the VM.  I was trying to do something similar with these names but can be improved on.